packages/fast-components-react-msft/app/color-picker.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import manageJss, { ComponentStyles, IJSSManagerProps, IManagedClasses } from "@microsoft/fast-jss-manager-react";
+import { IDesignSystem } from "@microsoft/fast-components-styles-msft";
+
+export interface IColorConfig {
+    foregroundColor: string;
+    backgroundColor: string;
+    accentColor: string;
+}
+
+export interface IColorPickerProps extends IColorConfig {
+    onColorUpdate: (colors: IColorConfig) => void;
+}
+
+export interface IColorPickerManagedClasses {
+    colorPicker: string;
+    colorPicker_label: string;
+}
+
+const styles: ComponentStyles<IColorPickerManagedClasses, IDesignSystem> = {
+    colorPicker: {
+        display: "flex",
+        height: "100%",
+        alignItems: "center"
+    },
+    colorPicker_label: {
+        margin: "0 12px"
```

Comment:
we can use the toPx function here

Reply:
I'd prefer to not  preform an operation if we don't need to.